src/coreclr/vm/syncblk.cpp

```diff
@@ -47,7 +47,7 @@ class  SyncBlockArray
 };
 
 // For in-place constructor
```

Comment:
This is an in-place constructor which is better to make sure it is aligned pointer-size.

Reply:
Hi, @jkotas 
Could you please review this PR?